services/src/main/java/org/keycloak/services/resources/admin/UsersResource.java

```diff
@@ -131,12 +136,17 @@ public Response createUser(final UserRepresentation rep) {
                 session.getTransactionManager().setRollbackOnly();
             }
             return ErrorResponse.exists("User exists with same username or email");
+        } catch (PasswordPolicyNotMetException e) {
+            if (session.getTransactionManager().isActive()) {
+                session.getTransactionManager().setRollbackOnly();
+            }
+            return ErrorResponse.error("Password policy not met for user " + username, Response.Status.BAD_REQUEST);
```

Comment:
@k-tamura I have some privacy/security concerns in returning the username in the response body. Maybe it is a better idea not to provide it there?

Reply:
@maksymgendin
Umm... I think there are no security/privacy problems but user name does not need to be included in response. So I will remove it if you have any concerns. 

@stianst @hmlnarik 
How do you think?

Reply:
@k-tamura My vote is also to not include username in the response